specification/contributors.md

```diff
@@ -2,88 +2,43 @@
 
 ### Maintainers
 
-Thanks to the following FOCUS Maintainers for their leadership and contributions to the FOCUS specification.
+Thanks to the following FOCUS Maintainers for their leadership and contributions to the FOCUS Release **v1.1** specification.
 
+* Alex Hullah (Goldman Sachs)
 * Christopher Harris (Datadog)
-* Dennis Park-Rodriguez (Amazon Web Services)
-* Erik Peterson (CloudZero)
 * Irena Jurica (Neos)
 * Joaquin Prado (FinOps Foundation)
 * Karl Kraft (Walmart)
 * Larry Advey (Twilio)
 * Michael Flanakin (Microsoft)
-* Mike Fuller (FinOps Foundation)
 * Riley Jenkins (Domo)
-* Rupa Patel (Google)
+* Rob Martin (FinOps Foundation)
+* Shawn Alpay (Envisor)
```

Comment:
Since Shawn changed companies/roles at the end of the 1.1 development cycle, which should be mentioned? I'd agree that Envisor makes sense since he was there during the majority of the development, but wanted to call it out for discussion.

Reply:
Ok, let's discuss it in tomorrow's Members' meeting.

Reply:
The group agreed to add Shawn on both groups: to Contributors and Maintainers.

Reply:
Clarity - we agreed he would have both companies listed. Provided suggestion and closing this conversation.